### PR TITLE
Update rules.tex

### PR DESCRIPTION
**Pull request by gngrbrdmn**
Checked grammar for up to line 150, might have gone overboard with edits.

### Please describe your change in one or two sentences

*Your description here*

### Please explain why do you think this change should be in the rules

*Your explanation here*

### CODE REVIEW

**Comment by mrshu** [comment]
Check the latest version of the [built PDF](https://github.com/RoboCupJuniorTC/soccer-rules/blob/gngrbrdmn-patch-1-pdf/tex/rules.pdf)!

File: tex/rules.tex [hunk 1]
```diff
@@ -93,27 +93,38 @@
...
+
+Robocup Junior Soccer consist of two sub-leagues:
+\textbf{``Soccer Open''} and \textbf{``Soccer Lightweight''}. These rules apply
+for both sub-leagues. There are two main differences between the two leagues.
+
+\textbf{``Soccer Lightweight''} is played using a special ball that emits an IR signal ball. Robots may weigh up to 1.1 kg, may have a ball-capturing zone of up to 2.5 cm, and may use batteries up to 15.0 nominal voltage.  
+
+\textbf{``Soccer Open''} is played using a passive, brighly colored orange ball. Robots may weigh up to 2.4 kg, may have a ball-capturing zone of up to 3.0 cm, and may use batteries up to 12.0 nominal voltage.  
+
+{please see 5. Ball, for balls specifications and 8. League
```
Comment (mrshu) [inline tex/rules.tex]: @gngrbrdmn I am not sure what you wanted to do here, but it seems that "please" should probably be "Please" here.

Let me know if there is some formatting option you wanted to use.

Thanks!

Comment (gngrbrdmn) [reply tex/rules.tex]: Yep.  Can you merge so that I can edit the file with the changes in it?

Comment (mrshu) [reply tex/rules.tex]: @gngrbrdmn You should be able to continue editing the file with current changes in it, that is:

https://github.com/RoboCupJuniorTC/soccer-rules/blob/gngrbrdmn-patch-1/tex/rules.tex

I am also happy to merge it though -- if you edit the file above, the changes will stay in this Pull Request, which is something I would personally prefer =)

**Pull request merged**